src/main/java/com/exercise/rest/MeasurementMedianResource.java

```diff
@@ -0,0 +1,27 @@
+package com.exercise.rest;
+
+import com.exercise.model.MeasurementMedian;
+import com.exercise.service.MeasurementMedianService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import java.sql.Timestamp;
+import java.util.Collection;
+
+@RequestMapping("/medians")
+@Controller
+public class MeasurementMedianResource {
+
+    @Autowired
+    private MeasurementMedianService measurementMedianService;
+
+    @RequestMapping(path = "{sensorId}/{fromTimestamp}/{toTimestamp}")
```

Comment:
Why did you choose a path variable for timestamps? Filters are usually passed as query parameters.